data/base-town.yaml

```diff
@@ -27,7 +27,7 @@ Crossing:
     id: 895
   deposit: &crossing_deposit
     id: 1900
-  exchange:
+  exchange: &crossing_exchange
```

Comment:
Why add this anchor when it isn't used?

Reply:
I was gonna use it for knife clan but decided not to as it wouldn't be needed.  Having the anchor there wouldn't impact anything by leaving it and I was being lazy.